Intersect.Client/Entities/Entity.cs

```diff
@@ -1386,6 +1386,66 @@ public virtual void DrawName(Color textColor, Color borderColor = null, Color ba
             );
         }
 
+        // NpcTags
+        public virtual void DrawNpcTag()
```

Comment:
Rename to `DrawTag` and add a `if (!Options.Npc.ShowNpcTags || entity is Player)` short-circuit check.